### PR TITLE
マイページの作成

### LINKED ISSUE #112: マイページの作成
マイページを作成し、作成したイベントと参加予定のイベントを表示する。

### PR DESCRIPTION
**Pull request by yh2020**
close #112

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 class UsersController < ApplicationController
   before_action :authenticate
 
+  def show
+    @user = User.where(id: current_user).first
+
+    @events_owner = Event.where(owner_id: current_user).where(Event.arel_table[:start_time].gteq Time.zone.now).order(:start_time)
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/NestedParenthesizedCalls: Add parentheses to nested method call `Event.arel_table[:start_time].gteq Time.zone.now`.
- Metrics/LineLength: Line is too long. [130/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+
+    @events_part = Event.where(Event.arel_table[:start_time].gteq Time.zone.now).joins(:tickets).merge(Ticket.where(user_id: current_user)).order(:start_time)
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/NestedParenthesizedCalls: Add parentheses to nested method call `Event.arel_table[:start_time].gteq Time.zone.now`.
- Metrics/LineLength: Line is too long. [158/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :users, controllers: { :omniauth_callbacks => "omniauth_callbacks" }
   root to: 'welcome#index'
-  get '/auth/:provider/callback' => 'sessions#create'
-  get '/logout' => 'sessions#destroy', as: :logout
+
+  devise_for :users, :controllers => {
```
Comment (ms2sato) [inline config/routes.rb]: **[rubocop]**
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

File: config/routes.rb [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
+    :omniauth_callbacks => "omniauth_callbacks"
```
Comment (ms2sato) [inline config/routes.rb]: **[rubocop]**
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

File: config/routes.rb [hunk 5]
```diff
@@ -1,8 +1,10 @@
...
+  }
+  resources :users, :only => :show
```
Comment (ms2sato) [inline config/routes.rb]: **[rubocop]**
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

**Comment by ms2sato** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[](https://www.sideci.com/github_repositories/49850663/auto_correct/rubocop/pull_request/113/branch/112-add-mypage)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.sideci.com/ja/docs/autofix-rubocop)

**Comment by yh2020** [comment]
Created Auto-correct Pull Request from [SideCI](https://www.sideci.com/)

[https://github.com/CircleAround/eventure/pull/114](https://github.com/CircleAround/eventure/pull/114)

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -1,6 +1,21 @@
 class UsersController < ApplicationController
   before_action :authenticate
 
+  def show
+    @user = User.where(id: current_user).first
+
+    @events_owner = Event.where(owner_id: current_user)
+    .where(Event.arel_table[:start_time]
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/MultilineMethodCallIndentation: Align `.where` with `.where` on line 7.

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -1,6 +1,21 @@
...
+    .gteq(Time.zone.now))
+    .order(:start_time)
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/MultilineMethodCallIndentation: Align `.order` with `.where` on line 7.

File: app/controllers/users_controller.rb [hunk 8]
```diff
@@ -1,6 +1,21 @@
...
+
+    @events_part = Event.where(Event.arel_table[:start_time]
+    .gteq(Time.zone.now))
+    .joins(:tickets)
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/MultilineMethodCallIndentation: Align `.joins` with `.where` on line 12.

File: app/controllers/users_controller.rb [hunk 9]
```diff
@@ -1,6 +1,21 @@
...
+    .merge(Ticket.where(user_id: current_user))
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/MultilineMethodCallIndentation: Align `.merge` with `.where` on line 12.

File: app/controllers/users_controller.rb [hunk 10]
```diff
@@ -1,6 +1,21 @@
...
+    .order(:start_time)
```
Comment (ms2sato) [inline app/controllers/users_controller.rb]: **[rubocop]**
- Style/MultilineMethodCallIndentation: Align `.order` with `.where` on line 12.

File: app/models/event.rb [hunk 11]
```diff
@@ -11,6 +11,9 @@ class Event < ActiveRecord::Base
   validates :end_time, presence: true
   validate :start_time_should_be_before_end_time
 
+  scope :before_now, -> { where("start_time < ?", Time.zone.now()) }
```
Comment (ms2sato) [inline app/models/event.rb]: **[rubocop]**
- Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)

File: app/models/event.rb [hunk 12]
```diff
@@ -11,6 +11,9 @@ class Event < ActiveRecord::Base
...
+  scope :after_now, -> { where("start_time >= ?", Time.zone.now()) }
```
Comment (ms2sato) [inline app/models/event.rb]: **[rubocop]**
- Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)

File: app/models/event.rb [hunk 13]
```diff
@@ -11,6 +11,9 @@ class Event < ActiveRecord::Base
   validates :end_time, presence: true
   validate :start_time_should_be_before_end_time
 
+  scope :before_now, -> { where("start_time < ?", Time.zone.now }
```
Comment (ms2sato) [inline app/models/event.rb]: **[rubocop]**
- unexpected token tRCURLY
  (Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

File: app/models/event.rb [hunk 14]
```diff
@@ -11,6 +11,9 @@ class Event < ActiveRecord::Base
...
+  scope :after_now, -> { where("start_time >= ?", Time.zone.now }
```
Comment (ms2sato) [inline app/models/event.rb]: **[rubocop]**
- unexpected token tRCURLY
  (Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

**Pull request merged**